apps/project/tests/test_schemas.py

```diff
@@ -1003,6 +1004,29 @@ def _query_check(project_id, **kwargs):
                 # It's inactive as ahead by Project.PROJECT_INACTIVE_AFTER_MONTHS
                 assert project.status == Project.Status.INACTIVE
 
+    def test_project_role(self):
+        query = '''
+            query MyQuery {
+              projectRoles{
+                id
+                level
+                title
+                type
+
+            }
+        }
+        '''
+        user = UserFactory.create()
+        # without login
+
+        self.query_check(query, assert_for_error=True)
+
+        # with login
+        self.force_login(user)
+        projectrole = ProjectRole.objects.count()
+        content = self.query_check(query)
+        self.assertEqual(len(content['data']['projectRoles']), projectrole)
```

Comment:
```suggestion
        project_role_count = ProjectRole.objects.count()
        
        # without login
        self.query_check(query, assert_for_error=True)

        # with login
        self.force_login(user)
        content = self.query_check(query)
        self.assertEqual(len(content['data']['projectRoles']), project_role_count)
```

Reply:
@thenav56 Can we make a factory for Roles and use that here?

Reply:
We have a static set of roles. For test, we have this function which generates here https://github.com/the-deep/server/blob/6a823454917cb15b7df7f4ee33552eaafe8bf027/utils/graphene/tests.py#L137-L189